project/src/pages/main/main.tsx

```diff
@@ -1,9 +1,18 @@
-import CitiCard from '../../components/citi-card/CitiCard';
+import CityCard from '../../components/city-card/CityCard';
 
 type MainScreenProps = {
   offersCount: number;
 }
 
+
+function renderCityCards(value: number, cityCard: JSX.Element) {
+  const arrayOfCards = [];
+  for (let i = 0; i < value; i++) {
+    arrayOfCards.push(<CityCard />);
```

Comment:
Забыл key указать, в целом это для статичного списка не так критично, но лучше не забывать
```suggestion
    arrayOfCards.push(<CityCard key={i} />);
```